Common/TableProducer/PID/pidTPCBase.cxx

```diff
@@ -48,16 +50,30 @@ struct PidMultiplicity {
     if (enableTable) {
       LOG(info) << "Table TPC PID Multiplicity enabled!";
     }
+    if (doprocessStandard == true && doprocessIU == true) {
+      LOG(fatal) << "Both processStandard and processIU are enabled, pick one!";
+    }
+  }
+
+  void processIU(aod::Collision const& collision, TrksIU const& tracksExtra)
```

Comment:
@jezwilkinson sorry, wrong line ;) I wanted to point you at this one

Reply:
Thanks @njacazio for the heads-up - just wondering, is there any way we are able to do this that avoids separate process functions? For TPCBase it's not such a big deal, since that's a very small and simple task, but I was looking into making the same Tracks->TracksIU change for the pidTPC/pidTPCFull table creators, and this gets messy to maintain if we have to duplicate the entire process function for just the table type

Reply:
Maybe we can merge, this PR in principle does not change the default behaviour